msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/AbstractAcquireTokenTest.java

```diff
@@ -68,6 +70,12 @@ public void setup() {
         AdbShellUtils.removePackage(AZURE_AUTHENTICATOR_APP_PACKAGE_NAME);
         AdbShellUtils.removePackage(COMPANY_PORTAL_APP_PACKAGE_NAME);
 
+        // CP may still be installed if device admin
+        if (CommonUtils.isPackageInstalled(COMPANY_PORTAL_APP_PACKAGE_NAME)) {
```

Comment:
nit: should we just do this (and get rid of line 71)?

Reply:
No, we don't want to be trying to (go through UI to) disable admin unless we know CP is an admin. This is a hack that says hey if we can't delete CP from ADB, then it is because CP is an admin -- let's disable admin try deleting again. 

AFAIK, there is no simple way to know if an app is an admin unless you are DPM.